### DISCUSSION TITLE
Use milestones to manage our iteration plans

### DISCUSSION THREAD

**Discussion by tyn1998** [hypertrons/hypertrons-crx#406] [General]
Small and quick iterations can keep our tempo.

<img width="1320" alt="image" src="https://user-images.githubusercontent.com/32434520/179646448-276d3f90-de2a-4c60-9eaa-b754f1544620.png">

The idea is borrowed from opensumi/core's [iteration plans](https://github.com/opensumi/core/milestones).

**Comment by tyn1998** [comment]
Just released v1.6.3, feel so good!

<img width="1221" alt="image" src="https://user-images.githubusercontent.com/32434520/180749903-a2352479-08b1-4352-8505-020fb0ac03fc.png">

**Reply by zhicheng-ning** [reply]
> feel so good!

When the task progress reaches 100%, the sense of achievement is comfortable

**Reply by tyn1998** [reply]
Yeah~~~
